test_utils/edx_platform_mocks/xmodule/modulestore/django.py

```diff
@@ -0,0 +1,17 @@
+"""Mock modulestore"""
+from opaque_keys.edx.keys import CourseKey
+
+
+class DummyModuleStore:  # pylint: disable=too-few-public-methods
+    """DUmmy module store class"""
+    def __init__(self):
+        self.ids = ['library-v1:org1+11', 'library-v1:org1+22']
+
+    def get_library_keys(self):
+        """mock modulestore library keys method"""
+        return [CourseKey.from_string(id) for id in self.ids]
```

Comment:
Mocking is our enemy in tests. Please keep close to the real result as much as possible
```suggestion
        return [LibraryLocator.from_string(id) for id in self.ids]
```
`from opaque_keys.edx.locator import LibraryLocator`